test/androidTest/java/org/thoughtcrime/securesms/NicknameEspressoTest.java

```diff
@@ -37,8 +37,37 @@ public void canSetNickname() {
             .goConversation()
             .goSettings()
                 .setNickname(testString)
+                .goConversation()
+            .goConversations()
+            .assertText(testString)
+            .goConversation()
+                .goSettings()
+                .resetNickname()
             .goConversation()
-                .assertText(testString);
+            .goConversations()
+                .assertNoText(testString);
```

Comment:
why is this test both setting and resetting the nickname? isn't it just supposed to set it?

Reply:
Ok sounds good!